managed-services/webhosting/how-to/order-dedicated-ip.mdx

```diff
@@ -0,0 +1,67 @@
+content:
+  h1: How to order a dedicated IP address for a Web Hosting plan
+  paragraph: This page explains how to order a dedicated IP address for a Web Hosting plan
+tags: webhosting
+dates:
+  validation: 2024-02-05
+  posted: 2023-12-20
+categories:
+  - managed-services
+---
+
+In shared Web Hosting, multiple websites share the same server and its resources. Each website typically has its own domain name, but they all use the same IP address for the server. Scaleway provides the option to purchase a dedicated IP address for your website, even within a shared hosting environment. 
+Some potential advantages of having a dedicated IP address for Web Hosting are:
+
+* **Email reputation:** A dedicated IP address can be beneficial for email reputation. If your shared server's IP address is associated with spam or malicious activities from other users, it could negatively impact your email deliverability. With a dedicated IP, your email reputation is separate from other users on the same server.
+* **SSL/TLS certificates:** If you want to secure your website with an SSL/TLS certificate, having a dedicated IP address can be necessary. While it is possible to use Server Name Indication (SNI) to enable SSL on shared IP addresses, not all older browsers support SNI. In such cases, a dedicated IP address becomes necessary for compatibility.
+* **Avoiding blacklisting:** Shared IP addresses can be more susceptible to being blacklisted if another user on the same server engages in spam or other malicious activities. With a dedicated IP, you have more control over the reputation of your IP address.
+
+<Message type="note">
+  Needing a dedicated IP address can depend on your specific requirements and the nature of your website. Many websites function perfectly well with shared IP addresses, and dedicated IPs are often more critical for specific scenarios, such as e-commerce websites that require SSL/TLS or those with unique server configurations.
+  Always assess your specific needs before deciding whether a dedicated IP is necessary for your shared hosting environment.
+</Message>
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
+  - You have a Web Hosting plan with a shared IP address
+</Message>
+
+## Activate a dedicated IP address for a Web Hosting plan
+
+1. Navigate to the **Managed Services** section of the [console](https://console.scaleway.com/) side menu and click **Web Hosting**. This action will lead you to the **Web Hosting** overview page.
+2. Click either <Icon name="seeMore" /> or click on the domain name associated with the Web Hosting you want to configure. This will take you to the **Hosting Information** page.
```

Comment:
```suggestion
2. Click either <Icon name="seeMore" /> or the domain name associated with the Web Hosting you want to configure. This will take you to the **Hosting Information** page.
```